sample/src/main/java/com/polidea/rxandroidble/sample/example1_scanning/ScanActivity.java

```diff
@@ -23,6 +28,8 @@
 
 public class ScanActivity extends AppCompatActivity {
 
+    public final static int PERMISSIONS_REQUEST_LOCATION = 16;
```

Comment:
line 31 and 32 are not needed as the permission request was extracted to a different PR